mne/tests/test_epochs.py

```diff
@@ -2049,6 +2049,14 @@ def test_concatenate_epochs():
     epochs2.event_id = dict(a=2)
     assert_raises(ValueError, concatenate_epochs, [epochs1, epochs2])
 
+    # check events are shifted, but relative position are equal
+    epochs_list = [epochs.copy() for ii in range(3)]
+    epochs_cat = concatenate_epochs(epochs_list)
+    for ii in range(3):
+        evs = epochs_cat.events[ii * len(epochs):(ii + 1) * len(epochs)]
+        rel_pos = epochs_list[0].events[:, 0] - evs[:, 0]
+        assert_true(sum(rel_pos - rel_pos[0]) == 0)
```

Comment:
@alexandrebarachant this works because you don't actually test for the absolute timings right? Would it make sense to write a test where the times are actually different? And what will this do if the order / timing is unknown?